### ISSUE TITLE
Is the only one object considered?

### ISSUE THREAD

**Issue by jakeoung** [JingchunCheng/SegFlow#5]
In DAVIS 2017 dataset, there are many objects in each image. I wondered that how we can deal with the different number of objects in cross entropy loss.
But it seems that in this work, only foreground and background is considered. Is it right?

**Comment by wasidennis** [comment]
Yes, we deal with foreground/background segmentation in this work on DAVIS 2016 dataset. If you would like to consider different object instances, you can try these papers in http://davischallenge.org/challenge2017/publications.html, including ours with the code at https://github.com/JingchunCheng/Seg-with-SPN.

In terms of the loss for multiple objects, one way we tried before is to set N channels in cross entropy loss for N objects in a video for training. Hope this will answer your question.
